bedrock/bower.json

```diff
@@ -16,36 +16,17 @@
   "dependencies": {
     "bedrock-angular": "^2.0.0",
     "bedrock-angular-alert": "^2.0.0",
-    "bedrock-angular-credential": "digitalbazaar/bedrock-angular-credential#master",
-    "bedrock-angular-identity-composer": "digitalbazaar/bedrock-angular-identity-composer#master",
+    "bedrock-angular-credential": "^1.0.0",
+    "bedrock-angular-identity-composer": "^1.0.0",
     "bedrock-angular-form": "^2.0.0",
-    "bedrock-angular-key": "digitalbazaar/bedrock-angular-key#master",
+    "bedrock-angular-key": "^1.0.0",
```

Comment:
This is outdated now. All of these versions probably need to be checked/updated to latest.